playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-prefix-check.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { expect } from '@playwright/test';
+import { test } from 'fixtures/dsm-fixture';
+import crypto from 'crypto';
+import * as mock from 'data/mock-address.json';
+import { KitTypeEnum } from 'dsm/component/kitType/enums/kitType-enum';
+import { SamplesNavEnum } from 'dsm/component/navigation/enums/samplesNav-enum';
+import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import { StudyNavEnum } from 'dsm/component/navigation/enums/studyNav-enum';
+import { Navigation } from 'dsm/component/navigation/navigation';
+import KitUploadPage from 'dsm/pages/kitUpload-page/kitUpload-page';
+import { KitUploadInfo } from 'dsm/pages/kitUpload-page/models/kitUpload-model';
+import { KitsColumnsEnum } from 'dsm/pages/kitsInfo-pages/enums/kitsColumns-enum';
+import KitsSentPage from 'dsm/pages/kitsInfo-pages/kitsSentPage';
+import KitsWithoutLabelPage from 'dsm/pages/kitsInfo-pages/kitsWithoutLabel-page';
+import ParticipantListPage from 'dsm/pages/participant-list-page';
+import ParticipantPage from 'dsm/pages/participant-page/participant-page';
+import ErrorPage from 'dsm/pages/samples/error-page';
+import FinalScanPage from 'dsm/pages/scanner-pages/finalScan-page';
+import TrackingScanPage from 'dsm/pages/scanner-pages/trackingScan-page';
+import { WelcomePage } from 'dsm/pages/welcome-page';
+import { logInfo } from 'utils/log-utils';
+import { waitForResponse } from 'utils/test-utils';
+
+/**
+ * Prefix check for Blood kit with Canada and New York address for LMS and Osteo2 studies.
+ *
+ * For Blood kit:
+ *  - Upload a kit with address for Canada or NY state
+ *  - Create kit label without PECGS prefix
+ *  - Kit will be on error page
+ *  - Tracking scan
+ *  - Final scan
+ */
+
+// don't run in parallel
+test.describe.serial('Kit Upload', () => {
+  let welcomePage: WelcomePage;
+  let navigation: Navigation;
+  let participantPage: ParticipantPage;
+  let shippingID: string;
+  let kitUploadInfo: KitUploadInfo;
+  let shortID: string;
+
+  const studies = [StudyEnum.LMS]; // StudyEnum.OSTEO2;
```

Comment:
Limit test for one study as we don't have limited amount of participants. One participant with too many kits can encounter error.